conf/added_to_team.st

```diff
@@ -1 +1,3 @@
-You have been added to team "$team_name$".
+You have been added to this team: "$team_name$".
+You now have access to this team's shared files, apps, and analyses.
```

Comment:
I wonder if it might be confusing to mention "files" here since we do not yet have the capability to share files/folders with groups (including teams).

Reply:
@tinalcv Based on our Zoom discussion, I think we want to rephrase this to just `You now have access to this team's shared apps.` until we have the work completed which would unlock data sharing with groups.

Reply:
```suggestion
You now have access to this team's shared apps.
```